pkg/model/config.go

```diff
@@ -97,6 +97,21 @@ func (c Config) VPCRef() (string, error) {
 	return c.VPC.RefOrError(c.VPCLogicalName)
 }
 
+func (c Config) VPCRefFromNetworkStack() (string, error) {
```

Comment:
Yeah, this is the result of setting `conf.VPC`: in the Network stack we do NOT want to import `${NetworkStackName}-VPC`, because that's what's happening now:

```
"VpcId": {
  "Fn::ImportValue": {
    "Fn::Sub": "${NetworkStackName}-VPC"
  }
}
```

By removing this else case from `c.VPC.RefOrError`:

```
else if i.IDFromFn != "" {
  return i.IDFromFn, nil
}
```

we can prevent this. Because our new VPCRefFromNetworkStack returns:

```
"VpcId": {
  "Ref": "VPC"
}
```

which is the desired behaviour for the Network Stack (because there VPC is defined).


Maybe we can merge some logic of the two functions (because now I introduced some duplication), do you have an opinion on this?